src/FunnelChart/FunnelSeries/FunnelAxis/FunnelAxisLabel.tsx

```diff
@@ -99,7 +132,12 @@ export const FunnelAxisLabel: FC<Partial<FunnelAxisLabelProps>> = ({
         fontFamily={fontFamily}
         fontSize={fontSize}
       >
-        {label}
+        {/* text wrapping isn't supported with svg's, so this split allows user to pass in a label with line breaks */}
+        {label.split('\n').map((line, index) => (
```

Comment:
Take a look at the treemap or bubble.

Reply:
oh found the existing `wrapText()` util in there, sweet!